docs/content/en/docs-v0.46.x/user-guide/managing-application/customizing-deployment/script-run.md

```diff
@@ -55,6 +55,47 @@ The commands run in the directory where this application configuration file exis
 
 ![](/images/script-run.png)
 
+### Execute the script file
+
+If your script is so long, you can separate the script as a file.
+You can put the file with the app.pipecd.yaml in the same dir and then you can execute the script like this.
+
+```
+apiVersion: pipecd.dev/v1beta1
+kind: KubernetesApp
+spec:
+  name: script-run
+  pipeline:
+    stages:
+      - name: SCRIPT_RUN
+        with:
+          run: |
+            sh script.sh
+```
+
+```
+.
+├── app.pipecd.yaml
+└── script.sh
+```
+
+## Builtin commands
+
+Currently, you can use the commands which are installed in the environment for the piped.
+
+For example, If you are using the container platform and the offcial piped container image, you can use the command below.
+- git
+- ssh
+- jq
```

Comment:
We don't install jq in 0.46.x images, because it's installed at 0.48.0 release
https://github.com/pipe-cd/pipecd/pull/5009

Reply:
fixed on [dc78b67](https://github.com/pipe-cd/pipecd/pull/5127/commits/dc78b676c422bce4c9189394734549ed1b878193)